.github/workflows/c3-e2e-dependabot.yml

```diff
@@ -0,0 +1,86 @@
+# Runs the C3 e2e tests on dependabot PRs
+
+name: C3 E2E Tests (Dependabot)
+
+on:
+  pull_request:
+    paths:
+      - packages/create-cloudflare/**
+
+env:
+  node-version: 18.17.1
+  bun-version: 1.0.3
+
+jobs:
+  get-dependabot-bumped-framework:
+    name: "Get bumped framework (dependabot-only)"
+    runs-on: ubuntu-latest
+    outputs:
+      bumped-framework-cli: ${{ steps.detect.outputs.result }}
+    if: |
+      github.event.pull_request.user.login == 'dependabot[bot]'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+          ref: ${{ github.head_ref }}
+      - name: Get PR description as string
+        id: get-pr-description
+        run: |
+          str=$(sed 's/`/\`/g' <<EOF
+            ${{ github.event.pull_request.body }}
+          EOF
+          )
+          echo 'result<<EOF' >> $GITHUB_OUTPUT
+          echo $str >> $GITHUB_OUTPUT
+          echo 'EOF' >> $GITHUB_OUTPUT
+      - name: detect-bumped-framework
+        id: detect
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            const json = require('./packages/create-cloudflare/src/frameworks/package.json')
+            const frameworkCliPackages = Object.values(json.frameworkCliMap);
+
+            const body = `${{ steps.get-pr-description.outputs.result }}`;
+
+            const semverRegexStr = '\\d+\\.\\d+\\.\\d+';
+
+            const frameworkCliRegex = new RegExp(
+              `(?:^|\\s+)Bumps\\s+\\[(${frameworkCliPackages.join(
```

Comment:
This feels rather fragile...

Reply:
Would we even notice if this stopped matching?

Reply:
> This feels rather fragile...

yeah I agree.... but since we're just taking the dependabot PR and inferring what's getting updated by its PR description there isn't anything much better that we can do...

alternatively I think we could get the last commit diff and from there detect what framework is getting bumped? I guess that should work and would be more robust right? 🤔 


> Would we even notice if this stopped matching?

no... yeah maybe we should fail the whole workflow if the match failed 🤔 (not it just silently continues)